test/service/test_request_validation.py

```diff
@@ -352,3 +352,8 @@ def test_order(self):
                     self.assertBadRequest(url, arg)
                 else:
                     self.assertResponseStatus(url, 200)
+
+    def test_version_param(self):
```

Comment:
```suggestion
    def test_version(self):
```

Reply:
https://github.com/DataBiosphere/azul/pull/4906/files#r1107557638

You thumbed this up but didn't address it.